src/Redux/fireRequest.tsx

```diff
@@ -158,9 +158,15 @@ export const fireRequest = (
                 msg: error.response.data.detail,
               });
             } else {
-              Notification.Error({
-                msg: "Something went wrong...!",
-              });
+              if (key === "operateAsset") {
+                Notification.Error({
+                  msg: "Wrong Middleware Hostname...!",
+                });
+              } else {
```

Comment:
This seems like a hack. Also, I thought we moved to useQuery. Please check `src/Utils/request/useQuery.ts`. If the middleware API calls still use fireRequest, please consider refactoring it to useQuery.

cc: @rithviknishad

Reply:
@khavinshankar the middleware api call uses fire request.. of course I can change it to useQuery if @rithviknishad agrees to it

Reply:
The existing usages of `operateAsset` was hard to switch to using useQuery/request as there were places where we were passing the redux's dispatch as params to the functions inside it.

However, switching to newer camera feed components would solve the issue as it is not using the deprecated `operateAsset`. (Eg: https://github.com/coronasafe/care_fe/pull/7654)

The newer camera feed components handles the error messages as expected and wouldn't have this underlying issue in the first place.

@hrit2773 Do you want to switch to using the newer camera feed components in the asset config page? If yes, you can make a separate PR.